app/scripts/modules/core/src/presentation/RenderOutputFile.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { dump as dumpYaml } from 'js-yaml';
+
+interface IRenderOutputFileProps {
+  outputFileObject: object;
+  options?: {
```

Comment:
should we omit the options? we only ever set them to `{ transformLinks: true }`

Reply:
if you think its a good idea

Reply:
it would certainly save 6 or 7 lines of code that we really can't afford

Reply:
it'll be more performant without all this cruft